mala/network/predictor.py

```diff
@@ -59,13 +59,6 @@ def predict_from_qeout(self, path_to_file, gather_ldos=False):
         predicted_ldos : numpy.array
             Precicted LDOS for these atomic positions.
         """
-        self.data.grid_dimension = self.parameters.inference_data_grid
-        self.data.grid_size = (
-            self.data.grid_dimension[0]
-            * self.data.grid_dimension[1]
-            * self.data.grid_dimension[2]
-        )
-
```

Comment:
Was this part of the code redundant? If I am not mistaken the grid dimension is relatively crucial for the calculation, but if deleting this does not break the class itself there must be some redundancy then?

Reply:
This is a code duplication, it fixes part of #508 .